client/src/app/shared/services/ingest.service.ts

```diff
@@ -209,25 +210,44 @@ export class IngestService {
       .pipe(map(data => data._embedded && data._embedded.projects ? data._embedded.projects[0] : null));
   }
 
-  public fetchSubmissionData(submissionId, entityType, filterState, params): Observable<PagedData<MetadataDocument>> {
+  public fetchSubmissionData(submissionId, entityType, params): Observable<PagedData<MetadataDocument>> {
     let url = `${this.API_URL}/submissionEnvelopes/${submissionId}/${entityType}`;
     const submission_url = `${this.API_URL}/submissionEnvelopes/${submissionId}`;
 
+    // IMPORTANT! The order of these if statements matters
+    // The url is overridden in each but params will be preserved.
+    // E.g. params.sort can be added and used in the url for filtering as well
+    // This might not be the best way to do this but with the varying endpoints, it's okay
```

Comment:
this would deserve some tests to make sure no one accidentally update the order and break it!